### PR TITLE
Allow opt out of import embedded schema file

### PR DESCRIPTION
**Pull request by bergmania**
This PR introduces  a new configuration so solutions or packages can opt-out of the import of embedded schema files.

To make this work non-breaking, I had to refactor our initialization of the StaticServiceProvider, as the IStarterFilters are executed after the hosted services. And thereby cannot be used for e.g. migrations.

### CODE REVIEW

File: src/Umbraco.Web.UI/Umbraco.Web.UI.csproj [hunk 1]
```diff
@@ -74,6 +74,7 @@
     <PackageReference Include="Umbraco.Code" Version="1.2.0">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Umbraco.TheStarterKit" Version="9.0.0" />
```
Comment (AndyButland) [inline src/Umbraco.Web.UI/Umbraco.Web.UI.csproj:77]: Was this intended to push to source control?

Comment (bergmania) [reply src/Umbraco.Web.UI/Umbraco.Web.UI.csproj:77]: Nope, thanks

Comment (bergmania) [reply src/Umbraco.Web.UI/Umbraco.Web.UI.csproj:77]:
```suggestion

```

File: src/Umbraco.Web.Common/Umbraco.Web.Common.csproj [hunk 2]
```diff
@@ -49,4 +49,8 @@
         </AssemblyAttribute>
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="DependencyInjection\UmbracoApplicationServicesCapture.cs" />
+    </ItemGroup>
+
```
Comment (bergmania) [inline src/Umbraco.Web.Common/Umbraco.Web.Common.csproj]:
```suggestion
    
```

**Review by AndyButland — APPROVED** [review]
Looks good to me and have tested with the option set and missing and we see the expected behaviour.

**Pull request merged**